chart/githubconnector/templates/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.name }}
+    chart: {{ include "github-connector-chart.chart" . }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.name }}
+    spec:
+      containers:
+      - name: {{ .Values.container.name }}
+        image: {{ .Values.container.image }}
+        resources:
+          limits:
+            memory: "128Mi"
+            cpu: "500m"
+        ports:
+        - containerPort: {{ .Values.container.containerPort }}
+        env:
+        - name: PORT
+          value: {{ toString .Values.container.containerPort | printf ":%s"}}
```

Comment:
GitHub Connector no longer uses a colon before port number, `{{ toString .Values.container.containerPort }}` would be enough. 

Please test this